client/src/actions/post.js

```diff
@@ -0,0 +1,95 @@
+import axios from 'axios'
+import { Toastr } from '../utils/toastr/Toastr'
+import { POSTS_END_LOADING, POSTS_RECIEVED, POSTS_START_LOADING } from '../utils/constants/actionsName'
+
+export const createPost = (uploadForm, setUploadForm) => {
```

Comment:
I'm not sure that createPost should be at action functions, cause it won't dispatch any action.

Reply:
Where do you suggest to keep it? I am not a big fan of placing it inside component. Open to discussion, though.

Reply:
utils/fileUpload  or something like that

Reply:
I am against putting it to utils. It's definitely not a helper function.